cloud/pkg/edgecontroller/controller/upstream.go

```diff
@@ -77,7 +76,7 @@ func SortInitContainerStatuses(p *v1.Pod, statuses []v1.ContainerStatus) {
 
 // UpstreamController subscribe messages from edge and sync to k8s api server
 type UpstreamController struct {
-	kubeClient   *kubernetes.Clientset
+	kubeClient   client.KubeEdgeClient
```

Comment:
Let's use `Get()` of informers in https://github.com/kubeedge/kubeedge/blob/master/cloud/pkg/edgecontroller/controller/upstream.go#L495-L534

Reply:
done